### DISCUSSION TITLE
Good first issue?

### DISCUSSION THREAD

**Discussion by dchirst** [developmentseed/titiler#615] [General]
Hi! 

I really enjoy Titiler (and its sibling projects). I'd love the opportunity to start contributing to it a little more. Is there a good first issue that I could help with, or a project board that I can take tasks from?

Thanks

**Comment by vincentsarago** [comment]
Hi @dchirst,
We very much welcome contributors 🙏 
There are couple `documentation` issues already opened and for the `code` part I don't think there is much for now. We don't really have roadmaps (I'm usually making update when we find bugs 😬) 

Depending on your skillset and what you want to do, the different `frontend` page could be better 🙈

**Reply by dchirst** [reply]
Thanks for the reply @vincentsarago . I'm happy to help on any of these tasks, including on other repos (tivt, cogeo-mosaic). I'll get started with a documentation task and go from there!

**Reply by vincentsarago** [reply]
🙏 I took the liberty to `re-open` the discussion if it can inspire other people to join the fun 😅
